src/utils/set-state.ts

```diff
@@ -0,0 +1,550 @@
+                [localTotalRewardsStorageKey, localTotalRewardsData],
+                [localTotalStakeStorageKey, localTotalStakeData]
+            ]);
+            console.log("OK: Successfully set vault reward.");
+        }
+    );
+
+
+async function main(): Promise<void> {
+    console.log("Initializing crypto functions...");
+    await cryptoWaitReady();
+
+    const keyring = new Keyring({ type: "sr25519" });
+    vault_1 = keyring.addFromUri(VAULT_1_URI);
+    sudoAccount = keyring.addFromUri(SUDO_URI);
+
+    console.log("Connecting to parachain...");
+    const api = await createSubstrateAPI(PARACHAIN_ENDPOINT);
+    console.log("OK: Successfully connected to parachain.\n");
+
+    const sudoAccountInterBtcApi = new DefaultInterBtcApi(api, "regtest", sudoAccount, ESPLORA_BASE_PATH);
+
+    const defaultVaultAccountId = vault_1.address;
+    const defaultCollateralSymbol = currencyIdToMonetaryCurrency(sudoAccountInterBtcApi.api.consts.currency.getRelayChainCurrencyId).ticker;
+    const defaultWrappedSymbol = sudoAccountInterBtcApi.getWrappedCurrency().ticker;
+    console.log(`Default vault to use: ${defaultVaultAccountId}-${defaultCollateralSymbol}-${defaultWrappedSymbol}.\n`);
+
+    const SET_VAULT_PARAMS_BASE = {
+        accountId: {
+            alias: "a",
+            type: "string",
+            demandOption: false,
+            describe: "accountId of the vault",
+            default: defaultVaultAccountId
+        },
+        collateralSymbol: {
+            alias: "c",
+            type: "string",
+            demandOption: false,
+            describe: "collateral currency symbol of the vault",
+            default: defaultCollateralSymbol
+        },
+        wrappedSymbol: {
+            alias: "w",
+            type: "string",
+            demandOption: false,
+            describe: "wrapped currency symbol of the vault",
+            default: defaultWrappedSymbol
+        }
+    };
+
+    try {
+        await yargs(hideBin(process.argv))
+            .command(
+                "balance",
+                "Sets token balance of an account",
+                {
+                    value: {
+                        alias: "v",
+                        type: "number",
+                        demandOption: true,
+                        describe: "new balance value"
+                    },
+                    currencySymbol: {
+                        alias: "c",
+                        type: "string",
+                        demandOption: true,
+                        describe: "currency symbol in which to set balance"
+                    },
+                    address: {
+                        alias: "a",
+                        type: "string",
+                        demandOption: false,
+                        describe: "address of which to set balance"
+                    }
+                },
+                setBalance(sudoAccountInterBtcApi)
+            )
+            .command(
```

Comment:
These commands tend to get large, it's fine for now but as a future improvement they can be refactored into their own separate JS modules: https://github.com/yargs/yargs/blob/main/docs/advanced.md#providing-a-command-module